lib/HomeSeerSystemObject.js

```diff
@@ -466,32 +466,43 @@ class HomeSeerSystem
 		// console.log(yellow("*Debug* - categories URL: " + this.network.categoriesURL.href ));
 		
 		var that = this;
-		// var getStatusInfo 	= await promiseHTTP({ uri:statusURL.href, json:true, strictSSL:false})
-		// var getControlInfo 	= await promiseHTTP({ uri:controlURL.href, json:true, strictSSL:false})
-		
-		var getStatusInfo = 	await fetch(statusURL).then ( response => response.json() );
-		var getControlInfo = 	await fetch(controlURL).then ( response => response.json() );
 
+		// wait for both fetches to finish before resolving the Promise
+		await Promise.all([fetch(statusURL), fetch(controlURL)])
+		.then(function (responses) {
+			// Get a JSON object from each of the responses
+			return Promise.all(
+			responses.map(function (response) {
+				return response.json();
+			})
+			);
+		})
+		.then(function (data) {
+			let getStatusInfo = data[0];
+			let getControlInfo = data[1];

```

Comment:
Don't these lines really just do the same operation as the preceding version which "await"-ed the result?  To me, it looks like the difference is that the two "await"s in my prior code execute sequentially -- i.e., the getStatusInfo  must fully complete, then the getControlInfo fully completes, then execution continues. Converting it into a Promise.all as you've done, allows the fetches to proceed in parallel, so it may be a bit faster overall, but otherwise doesn't change the basic operation. Am I missing something?